### DISCUSSION TITLE
Strange "None" key

### DISCUSSION THREAD

**Discussion by yuyou** [miguelgrinberg/python-socketio#752] [Q&A]
My code ran into this line and exception throws ""KeyError: None"".

https://github.com/miguelgrinberg/python-socketio/blob/a813bde0626c16fe693db74cbc2ea7c331a177d3/src/socketio/base_manager.py#L120

Could someone help explaining why using "None" as the key?

**Comment by miguelgrinberg** [comment]
What do you need to know? You are looking at an internal structure maintained by this package. The `None` there is correct.
